functions.php

```diff
@@ -1686,6 +1710,73 @@ function end_lvl(&$output, $depth) {
     $output .= "$indent</ul>\n";
   }
 }
+/// Data Classification definition using in Profile pages
+function data_classification_definition ($info){
+  $info = trim($info);
+  if ($info == "កាត់បន្ថយ")
+      $info = "Downsized";
+  else if ($info == "កាត់បន្ថយបន្ទាប់ពីដកហូត")
+      $info = "Downsized after revocation";
+  else if ($info == "គ្មានភស្តុតាងនៃការផ្លាស់ប្តូរ")
+      $info = "No evidence of adjustment";
+  else if ($info == "ដកហូត")
+      $info = "Revoked";
+  else if ($info == "ទិន្នន័យរដ្ឋាភិបាលពេញលេញ")
+      $info = "Government data complete";
+  else if ($info == "ទិន្នន័យរដ្ឋាភិបាលមិនពេញលេញ")
+      $info = "Government data partial";
+  else if ($info == "ទិន្នន័យដទៃទៀត")
+      $info = "Secondary source data";
+  else if ($info == "ទិន្នន័យបន្ទាប់បន្សំ")
+      $info = "Other data";
+
+  $info = strtolower(str_replace(" ", "_", $info));
+    echo '&nbsp; <div class="tooltip tooltip_definition ">';
+        if ($info != "" && $info!=__("Not found", "opendev"))
+          echo '<i class="fa fa-question-circle info-data-classification" title=""></i>';
+        if ($info == "no_evidence_of_adjustment"){
+          echo '<div class="tooltip-info tooltip-no_evidence_of_adjustment">';
+            echo '<p>'.__("ODC is not aware of any adjustments to the concession since it was first granted.", "opendev");
+            echo '</p>';
+          echo '</div>';
+        }else if ($info == "downsized"){
+          echo '<div class="tooltip-info tooltip-downsized">';
```

Comment:
I pushed the PO file into the language directory. If i'm not wrong the transifex has already pointed the source to file on the github.